docs/scripts/generate-previews.ts

```diff
@@ -24,6 +25,21 @@ import { getPagesManifest } from '../src/utils/getPagesManifest';
 import { META_INFO } from '../src/data/meta';
 import { SITE_NAME } from '../src/data/general';
 
+try {
+  registerFont(path.join(__dirname, `../public/fonts/Inter-Regular.otf`), {
```

Comment:
This will make sure the proper fonts get rendered when built by Amplify Hosting (or anyone really)